src/js/check/check.js

```diff
@@ -0,0 +1,22 @@
+
+const checkFirstValue = (digitElement,saveValue,oper,total)=>{
+  if(saveValue.split(oper)[0].length == 3)
+    alert('숫자는 최대 세자리까지만 입력 가능합니다.');
+  else{
+    total.innerHTML = saveValue.concat(digitElement.innerHTML);
+    saveValue = total.innerHTML;
+  }
+  return [saveValue,oper];
+}
```

Comment:
```suggestion
const checkFirstValue = (digitElement,saveValue,oper,total)=>{
  if(saveValue.split(oper)[0].length == MAX_DIGIT_LENGTH )
    alert(MESSAGE.INVALID_DIGIT_LENGTH);
  else{
    total.innerHTML = saveValue.concat(digitElement.innerHTML);
    saveValue = total.innerHTML;
  }
  return [saveValue,oper];
}
```

constant.js 파일에서 선언했던 여러 상수들을 활용하셔서 최대 입력 가능한 숫자의 수가 변동되어도 잘 동작하도록 바꾸면 더 좋은 코드가 될 것 같습니다!

Reply:
좋은 의견 감사합니다